js/build.js

```diff
@@ -3,27 +3,31 @@ Fliplet.Widget.instance('image-gallery', function (data) {
   var photoswipeTemplate = Fliplet.Widget.Templates['templates.photoswipe'];
   var wallSelector = '[data-image-gallery-id=' + data.id + '] .wall:not("[data-mce-bogus] [data-image-gallery-id=' + data.id + '] .wall")';
 
-  function initGallery() {
+  function initGallery(options) {
     var $wall = $(wallSelector);
     var $bricks = $();
+    options = options || {};
```

Comment:
@upplabs-alex-levchenko @YaroslavOvdii There should be a new line above and below a block of variable declarations, except when it's at the beginning or end of a `{}` enclosure.

```suggestion

    options = options || {};
```

Reply:
Fixed